.eslintrc.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
 	rules: {
 		'arrow-parens': [2, 'as-needed', { 'requireForBlockBody': true }],
 		'camelcase': [0],
-		'class-methods-use-this': [0],
+		'class-methods-use-this': [1],
```

Comment:
We can remove this line entirely and let the AirBnB defaults pick it up.

Reply:
updated in feature/cleanup-cleanup as this PR was already merged.

Reply:
lgtm in feature/cleanup-cleanup